vizro-ai/docs/pages/user-guides/run-vizro-ai-dashboard.md

```diff
@@ -0,0 +1,175 @@
+# How to run Vizro-AI dashboard
+
+This guide offers insights into different ways of running `VizroAI.dashboard` to generate Vizro dashboard.
+
+??? note "Note: API key"
+
+    Make sure you have followed the [LLM setup guide](../user-guides/install.md#set-up-access-to-a-large-language-model) and thatyour api key is set up in a `.env` file in the same folder as your Notebook file (`.ipynb`).
+
+## Run Vizro-AI dashboard
+
+### 1. Prepare the data and user prompt
+```py
+import vizro.plotly.express as px
+
+df = px.data.tips()
+
+user_question = """
+Create a one-page dashboard layout with the following components:
+
+1. Card:
+   - Position: Left of the page
+   - Size: Takes up 1/4 of the total page width
+   - Content: Display the text "This is Tips dataset"
+
+2. Table:
+   - Position: Right of the card
+   - Size: Takes up the remaining 3/4 of the page width
+   - Content: Display the Tips dataset
+"""
+```
+
+### 2. Generate and launch the dashboard
+```py
+from vizro_ai import VizroAI
+
+vizro_ai = VizroAI(model="gpt-4o-mini")
+dashboard = vizro_ai.dashboard([df], user_question)
+```
+This triggers the dashboard building process. Once Vizro-AI finishes the dashboard generation process, you can now launch the dashboard.
+
+!!! example "Generated dashboard"
+
+    === "Code"
+        ```py
+        import vizro.plotly.express as px
+        from vizro_ai import VizroAI
+        from vizro import Vizro
+
+        df = px.data.tips()
+        user_question = """
+        Create a one-page dashboard layout with the following components:
+
+        1. Card:
+           - Position: Left of the page
+           - Size: Takes up 1/4 of the total page width
+           - Content: Display the text "This is Tips dataset"
+
+        1. Table:
+           - Position: Right of the card
+           - Size: Takes up the remaining 3/4 of the page width
+           - Content: Display the Tips dataset
+        """
+        vizro_ai = VizroAI(model="gpt-4o-mini")
+        dashboard = vizro_ai.dashboard([df], user_question)
+        Vizro().build(dashboard).run()
+        ```
+
+    === "Result"
+        [![VizroAIDashboardPage1]][VizroAIDashboardPage1]
+
+
+    [VizroAIDashboardPage1]: ../../assets/user_guides/dashboard/dashboard2_page1.png
+
+## Retrieve the Python code of the dashboard
```

Comment:
we should explain that people still need to set the data manager

Reply:
Yeah, what is that about. I've seen it in the output code from Vizro-AI but haven't understood what to do. Please could you add a paragraph or let me know and I'll do it.